### ISSUE TITLE
Is an update coming for gnomAD v2.1.1 recently released by ANNOVAR?

### ISSUE THREAD

**Issue by vnttung** [WGLab/InterVar#38]
Hi, just as the tittle said, I was wondering if Intervar would work on the gnomAD v2.1.1 by ANNOVAR cause I've seen changes in the headers of the new files?

Thanks!

**Comment by quanliustc** [comment]
no, it is coming for updated clinvar and omim etc, not for gnomAD2
 will update with gnomAD 2.1 later.

**Comment by vnttung** [comment]
> no, it is coming for updated clinvar and omim etc, not for gnomAD2
> will update with gnomAD 2.1 later.

Thanks for the reply, looking forward to seeing the updates.

Best regards!
